src/bin/tikv-ctl.rs

```diff
@@ -716,9 +716,9 @@ impl DebugExecutor for Debugger {
         println!("all regions are healthy")
     }
 
-    fn remove_fail_stores(&self, store_ids: Vec<u64>) {
+    fn remove_fail_stores(&self, store_ids: Vec<u64>, region_ids: Option<Vec<u64>>) {
```

Comment:
Fun trick:

```rust
fn main() {
    // Avoid a `Some(_)`
    println!("{:?}", my_function(vec![true,false]));
    // You can still pass a `None`
    println!("{:?}", my_function(None));

}

// Note that `T` implements `Into<Option<T>>`.
fn my_function<MaybeVec>(my_value: MaybeVec) -> bool
where MaybeVec: Into<Option<Vec<bool>>> {
    // Get the value...
    let my_value = my_value.into()
        // Or a default!
        .unwrap_or(vec![true]);
    my_value.iter().any(|&x| x)
}
```

https://play.rust-lang.org/?gist=31b46eb4ad1513899e8be53d13732636&version=stable&mode=debug

Reply:
Nice trick. However in our code we get `regions` by
```rust
            let region_ids = matches.values_of("regions").map(|ids| {
                ids.map(|r| r.parse())
                    .collect::<Result<Vec<_>, _>>()
                    .expect("parse regions fail")
            });
```
It's already an `Option`. So, I prefer keep it.